src/main/java/com/sivalabs/springblog/comment/domain/CommentRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.sivalabs.springblog.comment.domain;
+
+import com.sivalabs.springblog.post.domain.PostEntity;
+import com.sivalabs.springblog.user.domain.UserEntity;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CommentRepository extends JpaRepository<CommentEntity, Long> {
+
+    // Finds all comments for a specific post.
+    List<CommentEntity> findByPost(PostEntity post);
```

Comment:
Let's add the custom methods while working on specific usecases. For now we can just create empty repositories.